tests/qcopy.test/runit

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 bash -n "$0" | exit 1
 
-set -e
+set -x
```

Comment:
I meant -e, but didn't account for cdb2sql failing because the db isn't up yet.   If you don't mind, I'll change this to expect cdb2sql to fail, but not the rest of the script.